packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -0,0 +1,199 @@
+import { E } from '@endo/eventual-send';
+import { TimeMath } from '@agoric/time';
+import { Far } from '@endo/marshal';
+import { natSafeMath } from '@agoric/zoe/src/contractSupport/index.js';
+import { makeTracer } from '@agoric/internal';
+import { AuctionState } from './util.js';
+
+const { Fail } = assert;
+const { subtract, multiply, floorDivide } = natSafeMath;
+
+const trace = makeTracer('SCHED', false);
+
+/**
+ * @file The scheduler is presumed to be quiescent between auction rounds. Each
```

Comment:
"presumed" huh?

How confident are we that this presumption cannot be violated?

If it is violated, what happens? How bad is it?